### DISCUSSION TITLE
Intégration MyFeelBack par défaut dans les solutions

### DISCUSSION THREAD

**Discussion by OlivierVS31** [AmauriC/tarteaucitron.js#882] [Ideas]
Bonjour,

Comme ma demande sur userpilot (dont j'attends les informations que vous avez demandées) je souhaiterais savoir si c'est possible aussi d'intégrer MyFeelBack (solution NPS et de questionnaire client) dans les solutions par défaut ?

Voici le lien vers leur politique de cookies : [MyFeelBack Politique de cookies](https://help.myfeelback.com/fr/quels-sont-les-cookies-d%C3%A9pos%C3%A9s-par-un-dispositif-de-collecte-myfeelback).

**Comment by AmauriC** [comment]
Bonjour,

Pour que je puisse l'intégrer à tarteaucitron il me faut le marqueur html que MyFellBack vous demande d'installer sur le site avec le détail des éventuelles options.

Merci!

**Reply by OlivierVS31** [reply]
@AmauriC c'est du tag de déclenchement dont on parle le voici (je l'ai anonymisé au niveau token) 👍 

`<script type='text/javascript'>
window._Mfb_useCookie = true;
// Review the commented lines. Insert real values for the placeholders
window._Mfb_ud = {
	_context : {
		lang : undefined, // You can force the language of the survey.
		privacyMode : false, 
		_page : {
			url : location.pathname,
			storageDuration : 30
			// You can add page's properties here ie : pageProperties : { property1 : 'value', property2 : 'value'}
		}
	}
};
(function() {
	var mfb = document.createElement('script');
	mfb.type = 'text/javascript';
	mfb.charset = 'UTF-8';
	mfb.async = true;
	mfb.id = 'MFBActor';
	mfb.src = 'https://actorssl-xxxxx.kxcdn.com/actor/xxxxxxxxxxxxxxxxxx/action';
	var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(mfb, s);
})();
</script>`

**Reply by AmauriC** [reply]
Merci, la suite sur https://github.com/AmauriC/tarteaucitron.js/issues/893
